analyzers/src/SonarAnalyzer.VisualBasic/Extensions/InvocationExpressionSyntaxExtensions.cs

```diff
@@ -20,13 +20,26 @@
 
 namespace SonarAnalyzer.Extensions
 {
-    public static class InvocationExpressionSyntaxExtensions
+    internal static class InvocationExpressionSyntaxExtensions
     {
         internal static bool IsMemberAccessOnKnownType(this InvocationExpressionSyntax invocation, string identifierName, KnownType knownType, SemanticModel semanticModel) =>
             invocation.Expression is MemberAccessExpressionSyntax memberAccess
             && memberAccess.IsMemberAccessOnKnownType(identifierName, knownType, semanticModel);
 
         internal static IEnumerable<ISymbol> GetArgumentSymbolsOfKnownType(this InvocationExpressionSyntax invocation, KnownType knownType, SemanticModel semanticModel) =>
             invocation.ArgumentList.Arguments.GetSymbolsOfKnownType(knownType, semanticModel);
+
+        internal static bool TryGetOperands(this InvocationExpressionSyntax invocation, out SyntaxNode leftOperand, out SyntaxNode rightOperand)
```

Comment:
I think you can just call the `out` variables  `left` and `right`, since the method is called `TryGetOperands`